lib/connect.js

```diff
@@ -289,7 +289,7 @@ function connect(token, options) {
  *   as a hint for variable bitrate codecs, but will not take effect for fixed
  *   bitrate codecs
  * @property {?string} [name=null] - Set to connect to a {@link Room} by name
- * @property {boolean} [networkQuality=false] - Whether to enable the Network
+ * @property {boolean|NetworkQualityConfiguration} [networkQuality=false] - Whether to enable the Network
```

Comment:
@syerrapragada Can you update the description here to describe the config object?